ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Feature Request
+about: Submit a request for a feature you would love to see implemented in Substra & help us improve :-)
+title: "[FEATURE_REQUEST]"
+labels: Type:Feature_Request
```

Comment:
There may be a syntax error here, it according to https://help.github.com/en/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository it should be 

```
labels: "Type:Feature_Request"
```

Actually since this is because `:` is a reserved yaml char, the smiley in about will create the same issue.

This also applies to the other templates that have the same issue.

Reply:
Thank you!!